source/ui/Components/Output/Console.qml

```diff
@@ -1,75 +1,73 @@
+            onDoubleClicked: {
+                if (outputComponent.getOutputItem(outputItemIndex)["textMode"] !== 0) {
+                    textarea.clear();
+                }
             }
-
         }
+        function clear() {
+            text = "";
+        }
+    }
 
-        Connections {
-            target: outputComponent
-            // Send when the memory changes (at any address).
-            onMemoryChanged: {
-                var _baseAddress = outputComponent.getOutputItem(outputItemIndex)["baseAddress"];
-                var _mode = outputComponent.getOutputItem(outputItemIndex)["textMode"];
-                // Check if the memory address that was changed (at least partly) belongs to
-                // the output item's source space.
-                if ((address+length) >= _baseAddress && _mode === 0) {
+    Connections {
+        target: outputComponent
+        // Send when the memory changes (at any address).
+        onMemoryChanged: {
+            var _baseAddress = outputComponent.getOutputItem(outputItemIndex)["baseAddress"];
+            var _mode = outputComponent.getOutputItem(outputItemIndex)["textMode"];
+            // Check if the memory address that was changed (at least partly) belongs to
+            // the output item's source space.
+            if ((address+length) >= _baseAddress && _mode === 0) {
+                it.updateContent(_baseAddress);
+            }else if(_mode !== 0 ){
+                if(address<=_baseAddress && address+length >= _baseAddress){
                     it.updateContent(_baseAddress);
-                }else if(_mode !== 0 ){
-                    if(address<=_baseAddress && address+length >= _baseAddress){
-                        it.updateContent(_baseAddress);
-                    }
                 }
             }
-            // Send when any item's settings where updated.
-            onOutputItemSettingsChanged: {
-                var mode = outputComponent.getOutputItem(outputItemIndex)["textMode"];
-                var baseAddress = outputComponent.getOutputItem(outputItemIndex)["baseAddress"];
-                if(mode === oldMode && baseAddress === oldAddress){
-                    return;
-                }
-                it.updateContent(outputComponent.getOutputItem(outputItemIndex)["baseAddress"]);
-                settingsWindowC.updateSettings();
+        }
+        // Send when any item's settings where updated.
+        onOutputItemSettingsChanged: {
+            var mode = outputComponent.getOutputItem(outputItemIndex)["textMode"];
+            var baseAddress = outputComponent.getOutputItem(outputItemIndex)["baseAddress"];
+            if(mode === oldMode && baseAddress === oldAddress){
+                return;
             }
+            it.updateContent(outputComponent.getOutputItem(outputItemIndex)["baseAddress"]);
+            settingsWindowC.updateSettings();
         }
     }
+
     function updateContent(_baseAddress) {
```

Comment:
Try out some characters (decimal code): 
- 27: Escape "" (greater equal sign)
- 7: Bell "" (°)
- 8: Backspace "" (plus-minus sign)
I am not sure if this has something to do with your code. In fact the correct ASCII values are returned but they are displayed the wrong way on my machine. Maybe this has something to do with the default font?